inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/iceberg/pom.xml

```diff
@@ -88,6 +88,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>sort-common</artifactId>
+            <version>1.8.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
I suggest add relocation if you import this dependency.
like sort-connector-base
```
      <relocation>
                                    <pattern>org.apache.inlong.sort.base</pattern>
                                    <shadedPattern>org.apache.inlong.sort.iceberg.shaded.org.apache.inlong.sort.base</shadedPattern>
                                </relocation>
```
Because some not changed connector won't be effected when modify sort-common and connector also dependency sort-common.

Reply:
Fixed

Reply:
> Fixed

@lordcheng10 I just use it as a example by relocation of sort-connector-base. you should shade sort-common